tensorflow/python/keras/layers/pooling.py

```diff
@@ -446,7 +446,12 @@ def __init__(self,
 @keras_export('keras.layers.AveragePooling2D', 'keras.layers.AvgPool2D')
 class AveragePooling2D(Pooling2D):
   """Average pooling operation for spatial data.
-
+  
+  >>> import numpy as np
```

Comment:
no need to import numpy if you are not using it

Reply:
ok. I forgot about that. I'll remove it